README.md

```diff
@@ -1,4 +1,4 @@
-# strip-comments [![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=W8YFZ425KND68) [![NPM version](https://img.shields.io/npm/v/strip-comments.svg?style=flat)](https://www.npmjs.com/package/strip-comments) [![NPM monthly downloads](https://img.shields.io/npm/dm/strip-comments.svg?style=flat)](https://npmjs.org/package/strip-comments) [![NPM total downloads](https://img.shields.io/npm/dt/strip-comments.svg?style=flat)](https://npmjs.org/package/strip-comments) [![Build Status](https://travis-ci.org/jonschlinkert/strip-comments.svg?branch=master)](https://travis-ci.org/jonschlinkert/strip-comments)
+# strip-comments [![NPM version](https://img.shields.io/npm/v/@freecodecamp/strip-comments.svg?style=flat)](https://www.npmjs.com/package/@freecodecamp/strip-comments) [![NPM monthly downloads](https://img.shields.io/npm/dm/@freecodecamp/strip-comments.svg?style=flat)](https://npmjs.org/package/@freecodecamp/strip-comments) [![NPM total downloads](https://img.shields.io/npm/dt/@freecodecamp/strip-comments.svg?style=flat)](https://npmjs.org/package/@freecodecamp/strip-comments)
```

Comment:
Should the name be?

# @freecodecamp/strip-comments

Also, I assume it's just because the package isn't available yet that the badges say invalid and package not found, it is public right?

Reply:
So you don't think it should be `# @freecodecamp/strip-comments` instead of `# strip-comments`?

`# @freecodecamp/strip-comments [![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=W8YFZ425KND68)`

Reply:
The convention seems to be to use the original name.  The `@freecodecamp` part is just the npm scope, after all.